client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-identify/index.tsx

```diff
@@ -41,7 +41,10 @@ export const Analyzer: FC< Props > = ( { onComplete, onSkip } ) => {
 	return (
 		<div>
 			<div className="import__heading import__heading-center">
-				<Title>{ translate( 'Where will you import from?' ) }</Title>
+				<Title>{ translate( 'Where is your current content or website?' ) }</Title>
```

Comment:
I feel like this could be simplified to "Where is your current website?" and be just as clear, but it's not a strong opinion.